bundles/org.openhab.binding.roku/src/main/java/org/openhab/binding/roku/internal/handler/RokuHandler.java

```diff
@@ -134,6 +140,22 @@ private void refreshPlayerState() {
         synchronized (sequenceLock) {
             String activeAppId = ROKU_HOME_ID;
             try {
+                if (thingTypeUID.equals(THING_TYPE_ROKU_TV)) {
+                    try {
+                        deviceInfo = communicator.getDeviceInfo();
+                        String powerMode = deviceInfo.getPowerMode();
+                        updateState(POWER_STATE, new StringType(powerMode));
+                        if (POWERON.equalsIgnoreCase(powerMode)) {
+                            updateState(POWER, OnOffType.ON);
+                        } else {
+                            updateState(POWER, OnOffType.OFF);
+                            return;
+                        }
+                    } catch (RokuHttpException e) {
+                        logger.debug("Unable to retrieve Roku device-info. Exception: {}", e.getMessage(), e);
```

Comment:
It seems redundant to log exception message when you are also logging full exception with stack trace:
```suggestion
                        logger.debug("Unable to retrieve Roku device-info.", e);
```